docs/welcome/quickstart.mdx

```diff
@@ -22,56 +22,21 @@ import {
   Button,
 } from "@chakra-ui/react";
 
-# Introduction
+## Step 1: API keys
```

Comment:
I think we should have the endpoints at the top so people are aware of what endpoints exist also the environments no? 

Additionally, do we want to add the quickstart repo + description to the top?

Reply:
The goal of this page is to get people having some sort of connection working ASAP. I don't think we should have all endpoints at the top, since it doesn't achieve the first goal.

Reply:
should we include quickstart repo then?

Reply:
It is there! In a tip. 👯